src/lib/alloc.c

```diff
@@ -812,13 +812,16 @@ static void *alloc_heap_buffer(struct mm_heap *heap, uint32_t flags,
 		platform_shared_commit(map, sizeof(*map));
 	}
 
-	/* size of requested buffer is adjusted for alignment purposes
-	 * since we span more blocks we have to assume worst case scenario
-	 */
-	bytes += alignment;
-
 	/* request spans > 1 block */
 	if (!ptr) {
+		/* size of requested buffer is adjusted for alignment purposes
+		 * since we span more blocks we have to assume worst case scenario
+		 */
+		bytes += alignment;
```

Comment:
Is this move a mere refactor? I can't see anything wrong with it either being or not being done...

Reply:
@paulstelian97 yes, a refactor. Reason: only make it where needed.